src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.NET.ILLink.targets

```diff
@@ -143,6 +150,13 @@ Copyright (c) .NET Foundation. All rights reserved.
       <TrimMode Condition=" '$(TrimMode)' == '' ">copyused</TrimMode>
     </PropertyGroup>
 
+    <!-- Set a default value for TrimmerRemoveSymbols unless set explicitly. -->
+    <PropertyGroup Condition=" '$(TrimmerRemoveSymbols)' == '' ">
+      <!-- The default is to remove symbols when debugger support is disabled, and keep them otherwise. -->
+      <TrimmerRemoveSymbols Condition=" '$(DebuggerSupport)' == 'false' ">true</TrimmerRemoveSymbols>
+      <TrimmerRemoveSymbols Condition=" '$(DebuggerSupport)' != 'false' ">false</TrimmerRemoveSymbols>
```

Comment:
TrimmerRemoveSymbols hooking up into linker will be done separately, right?

Reply:
It's already hooked up - `TrimmerRemoveSymbols` sets `_TrimmerLinkSymbols` (inverted) which is already consumed by the linker (`LinkSymbols` on the task -> `-b`). If you would like, I can change the linker task option (in a separate change) to `RemoveSymbols` so that we can simplify this.

Reply:
Simplifications in follow up PR sounds great